src/components/Preimage/PreImageCard.tsx

```
@@ -0,0 +1,138 @@
+					</div>
+
+					<div className='text-xs'>
+						<Address
+							variant='inline'
+							address={preimage.proposer}
+							truncateCharLen={4}
+						/>
+					</div>
+					<span className='whitespace-pre font-medium'>{preimage?.deposit && formatBnBalance(preimage?.deposit, { numberAfterComma: 2, withUnit: true }, network)}</span>
+					<div>
+						{preimage?.proposedCall && preimage?.proposedCall.section && preimage?.proposedCall.method && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)